Jenkinsfile

```diff
@@ -1,4 +1,10 @@
-#!/usr/bin/env groovy
```

Comment:
My editor would appreciate you not removing this 😢

Reply:
My mistake, syntax colors are back !